docs-chef-io/content/inspec/reporters.md

```diff
@@ -101,7 +101,23 @@ This is the basic text base report. It includes details about which tests passed
 
 ### json
 
-This reporter includes all information about the profiles and test results in standard json format.
+This reporter includes all information about the profiles and test results in standard json format. You may optionally pass through arbitrary structured JSON data via a json config with the `--config` parameter like the example below.
+
+Example config:
+
```

Comment:
```suggestion
This reporter includes all information about the profiles and test results in standard JSON format. You may optionally pass through arbitrary structured JSON data by setting a JSON configuration with the `--config` parameter. 

For example:

```

Reply:
That's fine with me @IanMadd. Committed and back to you for final review!

Reply:
Also updated sign-off for DCO. I guess don't use the Github UI if you care about that for Chef review process! 😆